CSS/QUIZPAGE/style.css

```diff
@@ -0,0 +1,70 @@
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 24px;
```

Comment:
You can use this shorthand `padding: 24px 10px 0;` if I am not wrong.

Reply:
Done.